README.md

```diff
@@ -2,7 +2,9 @@
 
 # Optuna: A hyperparameter optimization framework
 
+[![Python](https://img.shields.io/badge/python-3.5+-blue.svg)](https://www.python.org/download/releases/3.5.0/)
```

Comment:
I accessed the download URL ( https://www.python.org/download/releases/3.5.0/ ), but it was 404 page.

![image](https://user-images.githubusercontent.com/181413/69779019-ebd6da80-11e9-11ea-9404-130613a0502c.png)

Reply:
IMO, the URL can be replaced with a more general one (e.g., https://www.python.org/).
Optuna supports Python3.5.0 but it doesn't mean the specific version is more recommended than other supported versions.

Reply:
I really apologize for my late response and your comments totally make sense.
So, I made a change and now README of this branch looks like this.

![image](https://user-images.githubusercontent.com/16191443/70390674-86b48d80-1a10-11ea-823e-3eebf3bbb44c.png)

Here, I exclude 3.8 on purpose because some libraries do not fully support it as mentioned in https://github.com/optuna/optuna/pull/759.

Thanks.